testing/endpoint_v2.go

```diff
@@ -6,11 +6,35 @@ import (
 
 	"github.com/cosmos/gogoproto/proto"
 
+	abci "github.com/cometbft/cometbft/abci/types"
 	clientv2types "github.com/cosmos/ibc-go/v10/modules/core/02-client/v2/types"
 	channeltypesv2 "github.com/cosmos/ibc-go/v10/modules/core/04-channel/v2/types"
 	hostv2 "github.com/cosmos/ibc-go/v10/modules/core/24-host/v2"
 )
 
+// ParsePacketFromEventsV2 parses a packet from events using the encoded packet hex attribute.
```

Comment:
Please take a look at `testing/events.go`. There is a ParsePacketFromEvents there. This one should be moved there.